packages/google_maps_flutter/google_maps_flutter_web/lib/src/google_maps_controller.dart

```diff
@@ -194,9 +194,10 @@ class GoogleMapController {
   /// own configuration and are rendered on top of a GMap instance later. This
   /// happens in the second half of this method.
   ///
-  /// This method is eagerly called from the [GoogleMapsPlugin.buildView] method
-  /// so the internal [GoogleMapsController] of a Web Map initializes as soon as
-  /// possible. Check [_attachMapEvents] to see how this controller notifies the
+  /// This method is eagerly called from the
+  /// [GoogleMapsPlugin.buildViewWithConfiguration] method so the internal
```

Comment:
The implementation had already changed to `buildViewWithConfiguration`, this comment was just outdated.

Reply:
Good catch!